src/gui/advancedsettings.cpp

```diff
@@ -609,6 +621,25 @@ void AdvancedSettings::loadAdvancedSettings()
     // Announce to all tiers
     m_checkBoxAnnounceAllTiers.setChecked(session->announceToAllTiers());
     addRow(ANNOUNCE_ALL_TIERS, tr("Always announce to all tiers"), &m_checkBoxAnnounceAllTiers);
+
+#if (LIBTORRENT_VERSION_NUM >= 10101)
```

Comment:
qBittorrent does not support libtorrent <1.2.0 anymore, so this should be removed.

Reply:
I can do this tomorrow.